Get-TeamsStatus.ps1

```diff
@@ -66,7 +66,7 @@ $TeamsProcess = Get-Process -Name Teams -ErrorAction SilentlyContinue
 
 # Check if Teams is running and start monitoring the log if it is
 If ($null -ne $TeamsProcess) {
-    If($TeamsStatus -eq $null){ }
+    If ($null -eq $TeamsStatus) { }
```

Comment:
It's best practice in PowerShell to have the `$null` on the left of checks - See [here](https://docs.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-null?view=powershell-7.1#checking-for-null)